File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/wrapper/block/Block.java
package rbq.wtf.lycoris.client.wrapper.wrappers.wrapper.block;

import rbq.wtf.lycoris.client.wrapper.MapEnum;
import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapperClass;
import rbq.wtf.lycoris.client.wrapper.wrappers.wrapper.IWrapper;

@WrapperClass(mcpName = "net.minecraft.block.Block", targetMap = MapEnum.VANILLA189)
public class Block extends IWrapper {
    public Block(Object obj) {
        super(obj);
    }
}

File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/wrapper/block/Block.kt
package rbq.wtf.lycoris.client.wrapper.wrappers.wrapper.block

import rbq.wtf.lycoris.client.wrapper.MapEnum
import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapperClass
import rbq.wtf.lycoris.client.wrapper.wrappers.wrapper.IWrapper

@WrapperClass(mcpName = "net.minecraft.block.Block", targetMap = MapEnum.VANILLA189)
class Block(obj: Any) : IWrapper(obj)